### DISCUSSION TITLE
Would be nice to grab both thrust lever with mouse at the same time

### DISCUSSION THREAD

**Discussion by Hinkelstein2** [magknight/787aviatorsTracker#518] [Completed things] [answered]
**Is your feature request related to a problem? Please describe.**
I want control both thrust lever with the mouse at the same time.
In the moment i can only grab one by one. The cockpit would feel a little bit more complete.
Maybe reverser in the future too.

**Describe the solution you'd like**
Just have a look at the default 737 or zibo. Would be nice, if it it could work like this.

**Comment by aeroniemi** [comment, marked as answer]
Has been added in 1.6.8-beta11

**Reply by Hinkelstein2** [reply]
nice, thanks.
